### PR TITLE
Get rid of Ext warnings on console

### PR DESCRIPTION
**Pull request by annarieger**
With this MR some (annoying) ext-debug log messages on console will be avoided/fixed.

Also xtype of `momo.view.Component.Map` class was changed to match momo-workspace.

### CODE REVIEW

**Comment by dnlkoch** [comment]
Ready to merge!

File: app/Application.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ Ext.define('momo.Application', {
         var moduleUtil = momo.util.Module;
         var urlUtil = momo.util.URL;
 
+        // disable annoying debug messages from WAI-ARIA 1.0 recommendations.
+        Ext.enableAriaButtons = false;
+        Ext.enableAriaPanels = false;
```
Comment (marcjansen) [inline app/Application.js:34]: I am not sure, I generally strive for high accessibility. If this simply warns without much we can actually do to improve A11Y, then I am plus one. You decide @annarieger

File: classic/src/view/container/Viewport.js [hunk 2]
```diff
@@ -1,6 +1,12 @@
 Ext.define('momo.view.container.Viewport', {
     extend: 'Ext.container.Viewport',
 
+    requires: [
+               'BasiGX.*',
+               'GeoExt.*',
+               'momo.*'
```
Comment (marcjansen) [inline classic/src/view/container/Viewport.js:7]: Mhm… will this result in really all files being in the build? If so I am against this. If not, go ahead. Still feels messy as I think we shoudl declare our dependencies where we actually depend on them being present (i.e. in class files).

If you have a strong opinion on this go ahead and merge regardless of my headaches.

**Comment by marcjansen** [comment]
I have added two comments / qquestions. I'll leave it up to you whether you want to merge this. Thanks anyhow.

File: classic/src/view/container/Viewport.js (hunk 2 above)
Comment (dnlkoch) [inline classic/src/view/container/Viewport.js:7]: Yeah, we discussed this a lot of times. It's definitely not perfect, but I didn't see any other way as we declare our application context (and thereby the classes to load) dynamically.

**Comment by weskamm** [comment]
> Yeah, we discussed this a lot of times. It's definitely not perfect, but I didn't see any other way as we 
> declare our application context (and thereby the classes to load) dynamically.

What about http://docs.sencha.com/extjs/6.0/6.0.0-classic/#!/api/Ext-method-syncRequire ?

**Comment by marcjansen** [comment]
> >  Yeah, we discussed this a lot of times. It's definitely not perfect, but I didn't see any other way as we 
> > declare our application context (and thereby the classes to load) dynamically.
> 
> What about http://docs.sencha.com/extjs/6.0/6.0.0-classic/#!/api/Ext-method-syncRequire ?

Or the callback function for [`Ext.require()`](http://docs.sencha.com/extjs/6.0/6.0.0-classic/#!/api/Ext-method-require) ?

**Comment by weskamm** [comment]
Would be even nicer..

**Comment by dnlkoch** [comment]
I think both wouldn't work for (single file) production build or am I wrong here?

**Comment by marcjansen** [comment]
Not without more thinking and coding, I guess. This is probably best discussed somewhere else as we would need some additional logic.

**Comment by annarieger** [comment]
Thanks for your comments, guys. I think for now I'll leave `requires` as is. If it leads to big problems later after more application components were added, we'll come back to it and can try out other approaches.

Will merge now.

**Pull request merged**